src/assets/styles/main.scss

```diff
@@ -1,7 +1,138 @@
 html {
+  font-family: Freight, Avenir, Helvetica, Arial, sans-serif;
 }
 @import 'variables';
 @import 'nav';
 
+.home .site-header {
+  margin-bottom: 0 !important;
+}
+.wp-block-cover .wp-block-cover__inner-container p {
```

Comment:
All the wp-block classes can go in a seperate scss file called like '_blocks.scss'